c_plus_plus/src/model/audio/source_position.h

```diff
@@ -0,0 +1,23 @@
+#ifndef SOURCE_POSITION_H
+#define SOURCE_POSITION_H
+
+#include <QJsonValue>
+
+
+namespace Model
+{
+
+struct SourcePosition
+{
+    SourcePosition(double azimuth, double elevation);
+    ~SourcePosition();
+
+    static SourcePosition deserialize(const QJsonValue jsonValue);
+
+    double azimuth;
+    double elevation;
```

Comment:
Should does two member be public?

Reply:
Since it was just a simple structure, I thought it best to just leave them as public instead of creating getters/setters that would just make the file bigger than it should be. What do you think?

Reply:
I agree with you!